style.css

```diff
@@ -0,0 +1,3 @@
+header {
+  color: #808000;
+}
```

Comment:
- [ ] As required for the project your `style.css` file to include one `class` with styles for your header, kindly make sure you update your `<h1>` tag by adding a class that should be used to style your heading in the `style.css` file instead of using your `<header>` tag.